zone/raids.h

```diff
@@ -259,6 +295,9 @@ class Raid : public GroupIDConsumer {
 
 	RaidMember members[MAX_RAID_MEMBERS];
 	char leadername[64];
+	char MainAssisterPCs[MAX_RAID_MAIN_ASSISTERS][64];
```

Comment:
This needs to be `snake_case` eg `main_assist_pcs`

If you don't feel comfortable doing some of these casing refactors - @Kinglykrab or I can help do them since our IDE's will catch all of the references in code precisely

Reply:
All good.  Updates made and a few other spots caught